### DISCUSSION TITLE
Διαγραφή ημερών παρουσίασης?

### DISCUSSION THREAD

**Discussion by p14kairas** [courses-ionio/hci#1638] [Announcements]
Στο Caledly λέτε οτι "Αν δεν υπάρχουν διαθέσιμες ώρες μπορείτε να δοκιμάσετε στον ίδιο σύνδεσμο αύριο γιατί σταδιακά προσθέτουμε μέρες-ώρες ανάλογα με τις δυνατότητες-ανάγκες." 

Πριν μερικές μέρες λεγατε οτι θα μπορέσουμε να παρουσιάσουμε απο 22 Νοε - 03 Δεκ κανονικά μετά το αλλάξατε σε 22-24 και αυτή την στιγμή εγώ βλέπω μόνο Σήμερα Δευτέρα 22 σαν επιλογή . Για να έρθω στην Κέρκυρα πρέπει να κανονίσω το ταξίδι και δεν μπορώ να επιλέξω μέρα μέσα  σε 10 λεπτά και τώρα αντί να ανοίγουν μέρες κλείνουν τι γίνεται ακριβώς?

**Comment by epidrome** [comment]
Θα γίνουν παρουσιάσεις και Τρι-Τετ αλλά είχαν πολύ ζήτηση και έφυγαν αμέσως 

Για την τρέχουσα εβδομάδα δεν υπάρχουν άλλες διαθέσιμες ώρες εκτός από αυτές που φαίνονται

Κάποιες από τις αρχικές διαθέσιμες (40+) αυτής της εβδομάδας διαγράφηκαν (π.χ. Τρίτη 9-11πμ) γιατί δεν είχαν ζήτηση, η οποία είναι μικρότερη του αναμενόμενου και περίπου 20 αυτήν την στιγμή

Θα προσθέσουμε μέρες-ώρες για την επόμενη εβδομάδα και θα βγει ανακοίνωση προς το τέλος της εβδομάδας
